addon/initializers/embedded.ts

```diff
@@ -33,12 +33,32 @@ function normalizeConfig(userConfig:GivenConfig):ObjectConfig {
 
   if (configIsBoolean(userConfig)) {
     let embeddedConfig = { delegateStart: userConfig, config: {} }
-    deprecate('The `embedded` config property MUST be undefined or an an object', false, { id: 'bad-object-config', until: '1.0.0' })
+    deprecate(
+      'The `embedded` config property MUST be undefined or an an object',
+      false, {
+        id: 'ember-cli-embedded.bad-object-config',
+        until: 'not defined',
+        for: 'ember-cli-embedded',
+        since: {
+          enabled: '1.0.0'
```

Comment:
This deprecation warning appeared in the version `0.5.0`:
https://github.com/peopledoc/ember-cli-embedded/compare/v0.4.0...v0.5.0

```suggestion
          enabled: '0.5.0'
```

Also, it is useful to set `until: 'not defined',` if we have no information for that field? Why not removing it while we are at it?

Reply:
> Also, it is useful to set until: 'not defined', if we have no information for that field? Why not removing it while we are at it?

I forgot to answer myself: because the key `until` is mandatory.